VMEncryption/main/handle.py

```diff
@@ -788,7 +787,7 @@ def enable_encryption():
 
     ps = subprocess.Popen(["ps", "aux"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     ps_stdout, ps_stderr = ps.communicate()
-    if re.search(r"dd.*of={0}".format(disk_util.get_osmapper_path()), ps_stdout):
+    if re.search(r"dd.*of={0}".format(disk_util.get_osmapper_path()), ps_stdout.decode("utf-8")):
```

Comment:
adding comment:
# ps_stdout is data, so decode to string prior to regex for python2 and python3 compat